### ISSUE TITLE
Make failed

### ISSUE THREAD

**Issue by heatvent** [jameshilliard/WECB-NCS-GPL#2]
Trying to compile this on an ubuntu 12.04 VM with all pre-requisites installed.  It errors out ("Make failed") following the instructions in the readme file.  Not sure if anyone got this to work.

Thanks

**Comment by jameshilliard** [comment]
Yeah, don't think I tried compiling this but last I checked these actiontec GPL releases aren't really complete releases, they are usually missing various components needed to function properly.

**Comment by heatvent** [comment]
Not sure if I got lucky but trying this on Ubuntu 12.04.4 LTS 32-bit with no update / upgrade and installing the listed packages seemed to complete the build.  I was using 64-bit before and upgraded packages so not sure what the exact difference was.
